epam/cloud-pipeline
# Orchest tool

This PR provides new `Orchest` tool

Orchest is a web based tool for creating data science pipelines. Under the hood Orchest runs a collection of containers to provide a scalable platform that can run on your laptop as well as on a large scale cloud cluster.

Additional information and quick start tutorial could be found here: https://github.com/orchest/orchest

Steps to check that everything works fine:
- Start Orchest tool
- Wait until all internal containers will start and Orchest tool will be available on 8000 port (~ 3 minutes)
- Create test project as described in (https://orchest.readthedocs.io/en/stable/getting_started/quickstart.html)
- Create new pipeline 
- Wait until environment will be ready
- Create new step in the pipeline and open it with Jupiter
- Write down some code (some small one with any results that could confirm that Python interpreter is working e.g. `2 + 2` or `print("Hello")` )
- Go back to Orchest view 
- Run this step
- Go back in Jupiter view and check that results are available 

Also whole quick tutorial could be performed to check that everything works fine (https://orchest.readthedocs.io/en/stable/getting_started/quickstart.html)
